include/arbor/recipe.hpp

```diff
@@ -47,6 +47,14 @@ struct cell_connection {
     {}
 };
 
+struct gap_junction_connection {
+    cell_member_type location;
+    double ggap;
+
+    gap_junction_connection(cell_member_type loc, double g):
+            location(loc), ggap(g) {}
+};
+
 class recipe {
```

Comment:
Just a naming concern: 'location' can be a bit misleading, as we are here talking about an index into a series of end points; I would copy the nomenclature of `cell_connection`. Instead of `source` or `dest` we could use terms such as `peer` and `local`.

Reply:
Done as well